static-analysis/autodispose-lint/src/main/kotlin/com/uber/autodispose/lint/AutoDisposeIssueRegistry.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2018. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.uber.autodispose.lint
+
+import com.android.tools.lint.client.api.IssueRegistry
+import com.android.tools.lint.detector.api.CURRENT_API
+import com.android.tools.lint.detector.api.Issue
+import com.google.auto.service.AutoService
+
+@AutoService(IssueRegistry::class)
+class AutoDisposeIssueRegistry(override val issues: List<Issue> = listOf(AutoDisposeDetector.ISSUE)) : IssueRegistry() {
+  override val api: Int = CURRENT_API
+  override val minApi = 2
```

Comment:
Does it really need `minApi`? I've also noticed this in the checks from Zac

Reply:
With 26.3.0-beta04, the CURRENT_API actually points to AGP 3.3.0-alpha12+. Since this project still uses 3.2.1, we need `minApi = 2` so that it runs for `3.2.0-alpha7+`. We can remove it after we move to `3.3.0` stable.

Reply:
yeah I was doing this just going off of a thread I saw in lint-dev to basically support a lower lint version while still targeting newer APIs